cli/render.go

```diff
@@ -15,9 +24,134 @@ import (
 type RenderCommand struct {
 	*baseCommand
 	packConfig *cache.PackConfig
-	// renderOutputTemplate is a boolean flag to control whether the outpu
+	// renderOutputTemplate is a boolean flag to control whether the output
 	// template is rendered.
 	renderOutputTemplate bool
+	// renderToDir is the path to write rendered job files to in addition to
+	// standard output.
+	renderToDir string
```

Comment:
I'm not seeing where we add this to the ErrorContext. Am I just missing it in this view?  Would be useful to add if we are not yet.